app/home/route.ts

```diff
@@ -23,10 +22,11 @@ export default class Home extends Route {
       }
   }
 
-  @action
-  didTransition() {
+  buildRouteInfoMetadata() {
       const shouldShowVersionB = this.features.isEnabled(ABTesting.homePageHeroTextVersionB);
-      const version = shouldShowVersionB ? 'versionB' : 'versionA';
-      this.analytics.trackPage(undefined, undefined, undefined, version);
+      const abVersion = shouldShowVersionB ? 'versionB' : 'versionA';
+      return {
+          osfMetrics: { abVersion },
```

Comment:
This is more of an aside, but we aren't doing the AB testing of the logged out homepage anymore, are we?

Reply:
oh that's nice to know -- also i guess i kinda skimmed past this one because nothing is doing anything with that `osfMetrics.abVersion` -- will delete it 🎉